src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Server-side Sampling
+sidebar_order: 1
+description: "Learn about how you can control which transactions Sentry retains."
+---
+
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
+
+</Note>
+
+Capturing a single trace involves minimal overhead, but capturing traces for every page load or every API request may add an undesirable load to your system. Sampling your events allows you to better manage the number of events sent to Sentry, so you can tailor your volume to your organization's needs.
+
+_Server-side sampling_ lets you control which transactions Sentry retains by setting sampling rules and rates, so you see more of the transactions you want to explore further, and less of the ones you don’t. In contrast to [client-side sampling](/platform-redirect/?next=/configuration/sampling/), server-side sampling allows you to manage your sampling configuration centrally and lets you make changes without the need to reconfigure the SDK and redeploy your application.
+
+![The Server-side Sampling page with a list of rules](server-side-sampling.png)
+
+> [A trace](/product/sentry-basics/tracing/distributed-tracing/) represents the record of an entire operation you want to measure or track, like page load, an instance of a user completing some action in your application, or a cron job in your backend. A trace consists of one or more transactions and the trace can be distributed across multiple services or projects.
+
+Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more events from high converting pages, critical API endpoints, or need to focus on latency issues from your latest release.
+
+Server-side sampling also makes your SDK and your sampling configuration largely independent of each other. This helps if you're building native applications with many releases active at a given time.
+
+## How Sentry Samples
+
+Before understanding how Sentry samples, there are some concepts you need to know:
+
+- Total transaction volume: Total number of transactions the Sentry SDK has observed before any sampling has begun.
+- Client-side(SDK) sampling: The process of selecting which transactions are discarded or sent to Sentry for processing.
+- Server-Side sampling: The process of selecting which transactions are indexed after being processed.
```

Comment:
```suggestion
- Client-side(SDK) sampling: The process of selecting which transactions are discarded or sent to Sentry for processing.
- Server-side sampling: The process of selecting which transactions are indexed after being processed.
```

Reply:
If the two nouns connected by a hyphen is a Name of something then each word needs to be Capitalised.

Reply:
I get it mate but not sure if people will know about... its a minor thing but looks inconsistent

Reply:
Server-Side & Client-Side Sampling is also the finalised name, so let's only address any instances where it's a lower case s.

Reply:
ok in this case the Client-Side shall have a capitalized letter, shouldn't it?

Reply:
correct